examples/python-external-oauth/app.py

```diff
@@ -0,0 +1,70 @@
+from flask import Flask, render_template, url_for, redirect
+import requests
+from authlib.integrations.flask_client import OAuth
+
+app = Flask(__name__)
+oauth = OAuth(app)
+
+app.config["GITHUB_CLIENT_ID"] = "YOUR-GITHUB-CLIENT-ID"
```

Comment:
nitpick - load secrets from environment variables:

```
import os

...

app.config["GITHUB_CLIENT_ID"] = os.environ.get('GITHUB_CLIENT_ID')
```

Reply:
Nice suggestion, but we will keep it as is to make it simpler for the reader in the context of the guide 😄  

Thanks!